app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListOwnOfferActivity.java

```diff
@@ -12,28 +12,45 @@
 import ch.epfl.sweng.swenggolf.R;
 import ch.epfl.sweng.swenggolf.database.Database;
 import ch.epfl.sweng.swenggolf.database.ValueListener;
+import ch.epfl.sweng.swenggolf.profile.User;
 
 /**
  * Fragment which shows user own offers.
  */
 public class ListOwnOfferActivity extends ListOfferActivity {
+
+    private User user;
+
     @Override
     protected void prepareOfferData(View inflated,
                                     DatabaseOfferConsumer dbConsumer, List<Category> categories) {
+
         super.prepareOfferData(inflated, new DatabaseOfferConsumer() {
             @Override
             public void accept(Database db, List<Category> categories,
                                ValueListener<List<Offer>> listener) {
-                db.readOffers(listener, categories, Config.getUser().getUserId());
+                db.readOffers(listener, categories, user.getUserId());
             }
         }, categories);
     }
 
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstance) {
+
+        Bundle bundle = getArguments();
+        String title;
+
+        if (bundle != null) {
+            user = bundle.getParcelable("user");
```

Comment:
You can use the format ch.epfl.sweng.swenggolf.user so it has no risk to collide with other arguments. You can also use a constant to reduce the risk of typo in parcelable name.

Reply:
Thanks for the heads up.